less/home-page.less

```diff
@@ -1,13 +1,14 @@
 .home {
```

Comment:
its a good idea to give your outer container div a responsive margin so the text doesnt rub against the wall in the mobile view.